hack/quick_install.sh

```diff
@@ -30,7 +30,7 @@ while getopts ":hsr" option; do
    esac
 done
 
-export ISTIO_VERSION=1.16.2
```

Comment:
Was this intentional?

Reply:
yes, bumped up as this is the recommended version for kserve 0.11